lib/all/index.js

```diff
@@ -1,6 +1,11 @@
 import curry2 from '../_internal/_curry2'
-import toArray from '../toArray'
 
 export default curry2(function all(cb, data) {
-  return toArray(data).every(cb)
+  var i = 0
+  var length = data.length
+  while (i < length) {
+    if (!cb(data[i])) return false
+    i += 1
```

Comment:
replace `2` lines with `if (!cb(data[i++])) return false`
it will be shorter

Reply:
Ok. Also, I'll make performance test for it

Reply:
I also think it can be done like that:
```js
while (i < length && cb(data[i++])) {}
return i === length
```
but idk if it's shorter and efficient at all

Reply:
I've tested both variants:

<img width="400" alt="all - while" src="https://user-images.githubusercontent.com/25172855/39177249-6898956c-47b7-11e8-9570-903434ac7232.png">

<img width="400" alt="times_md_ ___documents_nanoutils" src="https://user-images.githubusercontent.com/25172855/39177254-6cfdec6a-47b7-11e8-92b1-76fb87882213.png">

I think it's because of short `i++` increment